.vscode/settings.json

```diff
@@ -12,5 +12,6 @@
   },
   "[yaml]": {
     "editor.defaultFormatter": "esbenp.prettier-vscode"
-  }
+  },
+  "workbench.colorCustomizations": {}
 }
```

Comment:
Probably irrelevant? :laughing:

Reply:
That can be removed. I was trying a new vscode extension.